cpp/src/strings/convert/convert_datetime.cu

```diff
@@ -551,7 +551,7 @@ struct check_datetime_format {
           break;
         }
         case 'S': {
-          auto const cv = check_value(ptr, item.length, 0, 60);
+          auto const cv = check_value(ptr, item.length, 0, 60);  // 60 occurs for leap seconds
```

Comment:
But this only allows `60` second value without verifying that it is a valid leap second, right?

Reply:
Leap second is not computable by comes from a fixed database as far as I know. This is the best we can do.

Reply:
I see. Thanks.

Reply:
I just realized that the doxygen in the header `convert_datetime.hpp` is still saying differently. Please also update it.

Reply:
Ah good catch. Thanks.

Reply:
In particular, `* | \%S | Second of the minute: 00-59 |`. This appears for both `is_timestamp` and `to_timestamp`. I'm not sure if we also need to update this for the `to_timestamp` API.

Reply:
Offline discussion we decided not to support leap-second in this API since it is not supported in Python, Java, and Spark. Although it is supported by std::chrono, it would not be possible to enforce without full support from cuda::std::chrono.

Reply:
@davidwendt if libcudf can access leap seconds data, will it be supported in future?

Reply:
I would say when libcu++ `chrono` functions replace this code in the future, we would support leap seconds then.